### DISCUSSION TITLE
Question: ***HOLY FUCK I'M GONNA LOSE*** A CHART I SPENT SO MANY HOURS ON WTF DO I DO - The question (Very urgent I'm gonna die please)

### DISCUSSION THREAD

**Discussion by Average-FNF-Modder** [FunkinCrew/Funkin#4044] [Q&A] [answered]
# What is your question?

I have been working on a chart for a fanmade Ugh Erect remix when randomly, for no god damn reason, the game just closed. After panicking for a moment I remembered that the game makes backups (Silly me), went to the folder and found a recent save of the chart. I was overjoyed until I actually tried loading it in. This has happened no other times with other charts but this time I had only made the chart to have the Erect variation, because that's all the song needed, so the chart had no default variation or any of it's files. And for a purpose unknown to me, the game requires there to be a metadata file for the default variation present or ***IT WON'T LOAD THE FILE!*** I have no way to directly edit the file other than going into Notepad++ which doesn't let me add files and stuff, just a wall of text, much of it looking like a garbled mess, and I scared for my life that I can't get this chart to load. Here is the window and the error message. I need some way to get this chart to load or I am jumping off a fucking bridge. If anyone knows how I can save this please lemme know. I also attached the .fnfc file in the best way I could manage as a Txt gotten from Notepad++ if that helps at all. Please for the love of anything if someone can help do it. I'm going to cry.

# Please

[chart-editor-2024-06-11-15-06-10.txt](https://github.com/user-attachments/files/15795646/chart-editor-2024-06-11-15-06-10.txt)

![image](https://github.com/FunkinCrew/Funkin/assets/157259571/3a241b9a-368f-48f6-959f-1cfbf79000a1)

![image](https://github.com/FunkinCrew/Funkin/assets/157259571/35c99076-a31b-4126-8397-efea18f6fdac)

**Comment by NotHyper-474** [comment, marked as answer]
Luckily for you .fnfc files are just renamed zip archives, so you can extract it and retrieve the files:
![image](https://github.com/FunkinCrew/Funkin/assets/40342021/857b6878-34d5-4745-bed9-edfd06c609b6)

You can just compress them back after editing, just make sure you're compressing to zip tho. From what I see you'll need not only the default metadata but also a chart file for it to load, in this case I just removed the second "erect" suffix from the files name and it worked. Voilà
![image](https://github.com/FunkinCrew/Funkin/assets/40342021/1dc9d73f-963a-4894-ae36-7eb4a740dadf)

**Reply by matolnz** [reply]
I have the same issue, just without any difficulty variants and idk what to do

**Comment by Average-FNF-Modder** [comment]
YOU'RE A FUCKING GODSEND I'M TRYING THIS RN HOLDUP.

**Comment by Average-FNF-Modder** [comment]
> Luckily for you .fnfc files are just renamed zip archives, so you can extract it and retrieve the files: ![image](https://private-user-images.githubusercontent.com/40342021/338790465-857b6878-34d5-4745-bed9-edfd06c609b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNjY2MjksIm5iZiI6MTcxODE2NjMyOSwicGF0aCI6Ii80MDM0MjAyMS8zMzg3OTA0NjUtODU3YjY4NzgtMzRkNS00NzQ1LWJlZDktZWRmZDA2YzYwOWI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEyVDA0MjUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmMGIxZTMxNTk0YjExZGNjNTY5OWFmYTQyYzg3NDhlOWY1YTE5NDI0MTU4MjBhZDAxMzBlZWUzYTcyZmU2NzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xMSpIikam4WwV5LmHfRHai-WjNmooK6qgp0sz1vnMlc)
> 
> You can just compress them back after editing, just make sure you're compressing to zip tho. From what I see you'll need not only the default metadata but also a chart file for it to load, in this case I just removed the second "erect" suffix and it worked. Voilà ![image](https://private-user-images.githubusercontent.com/40342021/338796004-1dc9d73f-963a-4894-ae36-7eb4a740dadf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNjY2MjksIm5iZiI6MTcxODE2NjMyOSwicGF0aCI6Ii80MDM0MjAyMS8zMzg3OTYwMDQtMWRjOWQ3M2YtOTYzYS00ODk0LWFlMzYtN2ViNGE3NDBkYWRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEyVDA0MjUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkyMTIxOTQ1MDA4YzMwOGVkZmFiYTE4MTE3N2Y1NDE4NmJmMDUyYTk3YWE2MTFhYWRjNTE1YTllNWI4YWEyMzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Jq9ROlXWtGjQ15zP0cjWXqq4e5BTL4goUeRC5GvF768)

## IT WORKED AND I WOULD JUST LIKE TO THANK GOD RQ AS AN ATHIEST. On a more serious note thank you I was actually about to cry ngl. This tip was a lifesaver and I would've been lost without it. Godspeed.
